### PR TITLE
Document 'disable_introspection' for GraphQL

### PR DESCRIPTION
**Pull request by thiaramus**
## Purpose of this pull request

This pull request (PR) is to document the missing env.php directive which was added to disable GraphQL introspection.

https://jira.corp.magento.com/browse/MC-37349
https://jira.corp.magento.com/browse/MC-37396

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.4/graphql/queries/

## Links to Magento source code

-  https://github.com/magento/graphql-ce/pull/308

whatsnew
Described how to disable [GraphQL introspection queries](https://devdocs.magento.com/guides/v2.3/graphql/queries/index.html#introspection-queries).

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/graphql/queries/index.md [hunk 1]
```diff
@@ -565,3 +565,13 @@ query IntrospectionQuery {
   }
 }
 ```
+
+### Disabling introspection querying
```
Comment (keharper) [inline src/guides/v2.3/graphql/queries/index.md:569]:
```suggestion
### Disable introspection querying
```

File: src/guides/v2.3/graphql/queries/index.md [hunk 2]
```diff
@@ -565,3 +565,13 @@ query IntrospectionQuery {
...
+
+Introspection querying is enabled by default. If you wish to disable it in production mode to improve security, please add the following to env.php.
```
Comment (keharper) [inline src/guides/v2.3/graphql/queries/index.md:571]:
```suggestion
Introspection querying is enabled by default. To disable it in production mode to improve security, add the following to your `env.php` file.
```

File: src/guides/v2.4/graphql/queries/index.md [hunk 3]
```diff
@@ -565,3 +565,13 @@ query IntrospectionQuery {
   }
 }
 ```
+
+### Disabling introspection querying
```
Comment (keharper) [inline src/guides/v2.4/graphql/queries/index.md:569]:
```suggestion
### Disable introspection querying
```

File: src/guides/v2.4/graphql/queries/index.md [hunk 4]
```diff
@@ -565,3 +565,13 @@ query IntrospectionQuery {
...
+
+Introspection querying is enabled by default. If you wish to disable it in production mode to improve security, please add the following to env.php.
```
Comment (keharper) [inline src/guides/v2.4/graphql/queries/index.md:571]:
```suggestion
Introspection querying is enabled by default. To disable it in production mode to improve security, add the following to your `env.php` file.
```

**Review by keharper — CHANGES_REQUESTED** [review]
@thiaramus I don't have permission to edit the content of your PR. Move this new section so that it appears before the "Example introspection queries". I think this is too important to be at the bottom of the page.

**Comment by thiaramus** [comment]
@keharper, I have an issue with my Adobe account which is being resolved at the moment. I have implemented the changes you suggested and will push them as soon as my account and permissions will be restored. Thank you!

File: src/guides/v2.3/graphql/queries/index.md (hunk 2 above)
Comment (rogyar) [reply src/guides/v2.3/graphql/queries/index.md:571]: I would strongly recommend specifying the full path to the configuration file. Very often only the filename brings a lot of confusion (happened to me as well in the documentation for 3rd party API system). There's a chance that a developer with no experience will create `env.php` file in the Magento root directory.

```suggestion
Introspection querying is enabled by default. If you wish to disable it in production mode to improve security, please add the following to `app/etc/env.php`.
```

Hope it makes sense

Comment (thiaramus) [reply src/guides/v2.3/graphql/queries/index.md]: @rogyar, good catch! I have corrected the path, hopefully it will add to the clarity!

**Review by keharper — APPROVED** [review]

**Comment by dobooth** [comment]
running tests

**Comment by dobooth** [comment]
running tests

**Pull request merged**